src/__tests__/AppContainer.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import 'jest-styled-components';
+import '@testing-library/jest-dom/extend-expect';
+import App from '../components/App/App.component';
+
+describe('validate app container works properly', () => {
+    test('render app contains elements', () => {
```

Comment:
The test description test doesn't match the test implementation, here the assertion is on a `CSS` class being present, not about some content being rendered.